pallet/ecdsa-authority/src/primitives.rs

```diff
@@ -103,13 +103,22 @@ pub struct Commitment {
 
 #[test]
 fn eth_signable_message() {
+	fn domain_separator(chain_id: u64, spec_name: &[u8]) -> Hash {
```

Comment:
Maybe we can change chain id to u32 this time.

Reply:
Make sense